lib/releases.sh

```diff
@@ -8,7 +8,14 @@ function get_latest_release() {
     release="$(curl -H "Authorization: token ${GITHUB_TOKEN}" -sL "${1}")" || ( set -x && exit 1 )
   fi
   # This gets the latest release as vx.y.z , ignoring any version with a suffix starting with - , for example -rc0
-  release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)] | max ")"
+  if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
```

Comment:
I don't think this is the correct variable, you want to know if you are on an Ubuntu OS, not if you are deploying an Ubuntu target cluster.

Reply:
the fact that we deploy Ubuntu on Ubuntu and Centos target cluster on CentOS should not be assumed. it could be changed

Reply:
Yes, I want to know OS flavor, because `sort -n +1` did not work for Centos, it was Ubuntu OS specific.

Reply:
What do you suggest, any pointers ?

Reply:
Centos has failed the test, with sort, so I was thinking it is OS dependant and I could use a condition. What kind of drawback do you foresee with these changes? i.e different OS(fedora, cinder) willn't work with above changes, is that what you are concerned about?

Reply:
I got your point now. What variable I could have use in that case, so even if we are not deploying Ubuntu on Ubuntu, it will be getting the correct sorting. Any ideas welcome

Reply:
check lib/common.sh . Everything you need is there. you could also have a look at 01_ that executes a script if it is Ubuntu and something else if CentOS

Reply:
Done